transforms/helpers/ast.ts

```diff
@@ -448,19 +440,13 @@ export function isDecoratorImportDeclaration(
   );
 }
 
-interface DecoratorImportDeclarationSource extends Literal {
-  value: string;
-}
+interface DecoratorImportDeclarationSource extends StringLiteral {}
```

Comment:
Either:

```suggestion
type DecoratorImportDeclarationSource = StringLiteral;
```

or make it a nominal type (a class with private fields works).

A nominal type might make sense here because you're doing additional checking that isn't represented
in the type signature.